<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>吔屎主程序</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/xiang/Pictures/556.JPG</normaloff>C:/Users/xiang/Pictures/556.JPG</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>191</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CommandLinkButton</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>CommandLinkButton</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="change">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>切换</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="timeEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>118</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QScrollBar" name="verticalScrollBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="keySequenceEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelGif">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../image/666.JPG</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_wang">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelmove">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>521</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>543</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="actionbetter"/>
    <addaction name="actionbest"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen">
   <property name="text">
    <string>good</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>well</string>
   </property>
  </action>
  <action name="actionbetter">
   <property name="text">
    <string>better</string>
   </property>
  </action>
  <action name="actionbest">
   <property name="text">
    <string>best</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
